pointgrey_camera_driver/debian/postinst

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+#DEBHELPER#
+
+# A udev rules file was added during install, load and trigger rules
+if [ "$1" = configure ]; then
+  if which udevadm > /dev/null; then
+    set +e # Disable exit on error
+    udevadm control --reload-rules
+    service udev restart
+    udevadm trigger
+    set -e # Re-enable exit on error
+  fi
+fi
```

Comment:
I did some testing, and ```udevadm control --reload-rules``` and ```service udev restart``` are not necessary. However, ```udevadm trigger``` is necessary to change the permissions of a connected device. Otherwise, a user would have to either reboot their computer or unplug and plug in the device.

Would you like me to reduce the script to just ```udevadm trigger```, or remove the ```postinst``` file entirely?